app/controllers/posts_controller.rb

```diff
@@ -1,44 +1,22 @@
 class PostsController < ApplicationController
-  # GET /posts
-  # GET /posts.json
   def index
-    @user = User.find_by(id: params[:id])
-    @posts = @user.posts
+    @user = User.includes(:posts).find_by(id: params[:id])
+    # @posts = @user.posts
```

Comment:
- [ ] I suggest that you remove any unused and commented code from your project, as this can make the code look cluttered and add extra lines of whitespace. If the comments are necessary and descriptive, feel free to leave them; otherwise, delete them.